Unix/base/oiomi.h

```diff
@@ -174,683 +165,689 @@ FILE_EVENT0(20020, trace__DispatchAssocReq_OutOfMemory_Impl, LOG_ERR, PAL_T("_Di
 #endif
 FILE_EVENT1(20021, trace_HTTP_RequestAllocFailed_Impl, LOG_ERR, PAL_T("HTTP: Error on allocating HttpRequest(%p)"), void *)
 #if defined(CONFIG_ENABLE_DEBUG)
+#define trace_HTTP_ClientAuthFailed(a0, a1) trace_HTTP_ClientAuthFailed_Impl(__FILE__, __LINE__, scs(a0), scs(a1))
+#else
+#define trace_HTTP_ClientAuthFailed(a0, a1) trace_HTTP_ClientAuthFailed_Impl(0, 0, scs(a0), scs(a1))
+#endif
+FILE_EVENT2(20022, trace_HTTP_ClientAuthFailed_Impl, LOG_ERR, PAL_T("HTTP: Client Authorization failed. gss:(%s) mech:(%s)"), const char *, const char *)
+#if defined(CONFIG_ENABLE_DEBUG)
 #define trace_HttpSocket_CannotPostMessage(a0, a1, a2, a3) trace_HttpSocket_CannotPostMessage_Impl(__FILE__, __LINE__, a0, a1, a2, a3)
 #else
 #define trace_HttpSocket_CannotPostMessage(a0, a1, a2, a3) trace_HttpSocket_CannotPostMessage_Impl(0, 0, a0, a1, a2, a3)
 #endif
-FILE_EVENT4(20022, trace_HttpSocket_CannotPostMessage_Impl, LOG_ERR, PAL_T("HttpSocket: %p Cannot posting message %p for interaction [%p]<-%p"), void *, Message *, Interaction *, Interaction *)
+FILE_EVENT4(20023, trace_HttpSocket_CannotPostMessage_Impl, LOG_ERR, PAL_T("HttpSocket: %p Cannot posting message %p for interaction [%p]<-%p"), void *, Message *, Interaction *, Interaction *)
```

Comment:
I don't think event numbers should be modified. Clients use those to filter logs.